dev/com.ibm.ws.jpa_locking_fat/publish/servers/EntityLocking/database.xml

```diff
@@ -0,0 +1,39 @@
+<!--
+    Copyright (c) 2020 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+ <server>
```

Comment:
It looks like this database.xml isnt included in the server.xml. Do you think server.xml should be updated to include this file? or should this just be deleted?